SafeExamBrowser.Browser.Contracts/SafeExamBrowser.Browser.Contracts.csproj

```diff
@@ -75,6 +75,10 @@
       <Project>{30b2d907-5861-4f39-abad-c4abf1b3470e}</Project>
       <Name>SafeExamBrowser.Settings</Name>
     </ProjectReference>
+    <ProjectReference Include="..\SafeExamBrowser.UserInterface.Contracts\SafeExamBrowser.UserInterface.Contracts.csproj">
```

Comment:
Right, makes sense. I thought it wouldn't be a problem to use `.Contracts` assemblies, since they only contain interface definitions and have no reference to an actual UI assembly (like WinForms or WPF).

Solved it by moving the event handler to `SafeExamBrowser.Browser.Contracts`, which already contains similar event handler definitions.

Reply:
Excellent, that's perfect.